cds/modules/fixtures/data/pages/guides/search.html

```diff
@@ -0,0 +1,398 @@
+
+<h3 id="advancedconcepts">Advanced concepts</h3>
+
+<h4 id="boosting">Boosting</h4>
+
+<p>You can use the boost operator <code>^</code> when one term is more relevant than another. For instance, you can search for all records with the phrase <em>Dark matter</em> in either <em>title</em> or <em>description</em> field, but rank records with the phrase in the <em>title</em> field higher:</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=title.title:%22dark%20matter%22%5E3%20description:%22dark%20matter%22"><code>title:"dark matter"^5 description:"dark matter"</code></a></p>
+
+<p>Note: <code>^</code> is followed by a positive floating point number specifying the boost value compared to the default 1.
+
+<h4 id="fuzziness">Fuzziness</h4>
+
+<p>You can search for terms similar to but not exactly like your search term using the fuzzy operator <code>~</code>.</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=drak~"><code>drak~</code></a></p>
+
+<p>Results will match records with terms similar to <code>drak</code> which would e.g. also match <code>dark</code>.</p>
+
+<h4 id="proximitysearches">Proximity searches</h4>
+
+<p>A phrase search like <code>"president visit"</code> by default expect all terms in exactly the same order, and thus for instance would not match a record containing the phrase <em>"Visit of his excellency the President of..."</em>. A proximity search allows that the terms are not in the exact order and may include other terms inbetween. The degree of flexiblity is specified by an integer afterwards:</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=%22president%20visit%22~10"><code>"president visit"~10</code></a></p>
+
+<h4 id="wildcards">Wildcards</h4>
+
+<p>You can use wildcards in search terms to replace a single character (using <code>?</code> operator) or zero or more characters (using <code>*</code> operator).</p>
+
+<p><strong>Example:</strong> <a href="https://videos.cern.ch/search?page=1&size=21&q=title.title:(crab%20cavit*)"><code>title.title:(crab cavit*)</code></a></p>
+
+The result will match videos with title referring to cavities also in languages other than english.
+
+<p>Wildcard searches can be slow and should normally be avoided if possible.</p>
+
+<hr />
+
+<h2 id="fieldsreference">Fields reference</h2>
+
+<p>The table below lists the data type of each field. Below is a quick description
+of what each data type means and what is possible.</p>
+
+<ul>
+<li><strong>string</strong>: Field does not require exact match (example field: <code>title.title</code>).</li>
+
+<!--
+
+<li><strong>string (exact)</strong>: Field requires exact match (example field: <code>doi</code>).</li>
+
+<li><strong>CV</strong>: Controlled vocabularies are similar to "string (exact)" but takes only a specific list of values.</li>
```

Comment:
The description is missing this bit, I think we should put the explanation of what CV field type is, otherwise it is not clear, especially for the language